test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamplesDynamicApi.cs

```diff
@@ -382,7 +382,7 @@ private void CategoricalFeaturizationOn(params string[] dataPath)
                 .Append(mlContext.Transforms.Categorical.OneHotEncoding("CategoricalFeatures", "CategoricalBag", OneHotEncodingTransformer.OutputKind.Bag))
                 // One-hot encode the workclass column, then drop all the categories that have fewer than 10 instances in the train set.
                 .Append(mlContext.Transforms.Categorical.OneHotEncoding("Workclass", "WorkclassOneHot"))
-                .Append(new CountFeatureSelector(mlContext, "WorkclassOneHot", "WorkclassOneHotTrimmed", count: 10));
+                .Append(mlContext.Transforms.FeatureSelection.CountFeatureSelectingEstimator("WorkclassOneHot", "WorkclassOneHotTrimmed", count: 10));
```

Comment:
>Estimator [](start = 83, length = 9)

why everything in this examples doesn't have estimator name, and you code has?

Reply:
That's a good question, we need to see whether we want the extensions to ml context to be named like the dynamic API or if they should be called like the static extensions. In my opinion it makes more sense to have them called like the dynamic API, as they call the dynamic API. But it is to be decided. I have seen both things. 

---
In reply to: [235509164](https://github.com/dotnet/machinelearning/pull/1683#discussion_r235509164) [](ancestors = 235509164)

Reply:
@sfilipi what is the general policy?

---
In reply to: [235540321](https://github.com/dotnet/machinelearning/pull/1683#discussion_r235540321) [](ancestors = 235540321,235509164)